### PR TITLE
Insight review

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Some comments from SensioLabs Insight platform.

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
ping @eric-chau can I merge ?

File: NestedNode/Repository/NestedNodeRepository.php [hunk 1]
```diff
@@ -177,7 +177,7 @@ public function insertNodeAsLastChildOf(AbstractNestedNode $node, AbstractNested
      * @return \BackBee\NestedNode\AbstractNestedNode             The inserted node
      * @throws \BackBee\Exception\InvalidArgumentException Occurs if the node is an ancestor of $parent or $parent is not flushed yet
      */
-    protected function _insertNode(AbstractNestedNode $node, AbstractNestedNode $parent, $newLeftNode)
+    protected function insertNode(AbstractNestedNode $node, AbstractNestedNode $parent, $newLeftNode)
```
Comment (eric-chau) [inline NestedNode/Repository/NestedNodeRepository.php]: Care if you rename a protected method you do a BC.

**Comment by eric-chau** [comment]
:+1: good job thank you @crouillon

**Comment by mickaelandrieu** [comment]
> :+1: good job thank you @crouillon
I'm @mickaelandrieu but thanks !

**Comment by eric-chau** [comment]
XD soz dude XD

**Comment by mickaelandrieu** [comment]
I wont fixup in case we have unexpected bugs ;)

**Comment by eric-chau** [comment]
ok

**Pull request merged**